tools/java/org/unicode/cldr/util/data/langtagRegex.txt

```diff
@@ -75,12 +75,12 @@ $langtag = (?: ( $language )
       (?: $s ( $privateUse ) )? 5%);
 
 # Here is the final breakdown, with capturing groups for each of these components
-# The variants, extensions, grandfathered, and private-use may have interior '-'
+# The variants, extensions, legacy, and private-use may have interior '-'
```

Comment:
I think this is specifically referring to the $grandfathered expression here, not sure we want to change this unless we also change that.

Reply:
There are two .java files that mention this file name, and neither explicitly looks for or mentions "grandfathered".